### DISCUSSION TITLE
Cutelyst directly in QT

### DISCUSSION THREAD

**Discussion by ricardopesce** [cutelyst/cutelyst#319] [Q&A]
Hello. Would there be some simple explanation about how to use cutelyst directly from the QTCreator? I don't know how to include cutelyst in QTCreator's new project creation.

**Comment by dantti** [comment]
Simply copy the [cutelyst](https://github.com/cutelyst/cutelyst/tree/master/qtcreator/cutelyst) folder into `~/.config/QtProject/qtcreator/templates/wizards/qtcreator/`
